velox/connectors/hive/HiveDataSink.h

```diff
@@ -541,6 +541,8 @@ class HiveDataSink : public DataSink {
   void closeInternal();
 
   const RowTypePtr inputType_;
+  // Written data columns into file.
+  RowTypePtr dataType_;
```

Comment:
I see what Masha suggested was
> store indices of these in a member variable

Guess that should look more like the partitionChannels_ data member and might work better. Otherwise you still need to compute the indices of the child vectors of the data input every time you call makeDataInput(), as you're doing it right now.

Reply:
@gggrace14 Added dataChannels_ member to store the indices of data columns.